packages/crypto/src/networks/devnet/milestones.json

```diff
@@ -51,5 +51,11 @@
     {
         "height": 1750000,
         "vendorFieldLength": 255
+    },
+    {
+        "height": 1895000,
```

Comment:
I think we should reduce the milestone to `1840000` then we have about a week before it kicks in.

Reply:
Done in a8b40858d.